### PR TITLE
fix(core): Allow all executions to be stopped

### PR DESCRIPTION
**Pull request by flipswitchingmonkey**

### CODE REVIEW

File: packages/cli/src/Server.ts [hunk 1]
```diff
@@ -1246,7 +1246,7 @@ export class Server extends AbstractServer {
 					throw new ResponseHelper.NotFoundError('Execution not found');
 				}
 
-				const execution = await Db.collections.Execution.findOne({
+				const execution = await Db.collections.Execution.exist({
```
Comment (krynble) [inline packages/cli/src/Server.ts:1249]: I'd suggest changing only this file.

Instead of making this change here revert the change to be `findOne` instead of `exists`.

Then, below, after the check for queue mode where we do `const result = await this.activeExecutionsInstance.stopExecution(executionId);` we can then have a check such as:

```
const result = await this.activeExecutionsInstance.stopExecution(executionId);
				let returnData: IExecutionsStopData;
				if (result === undefined) {
					if (['running', 'new'].includes(execution.status)) {
            // Do the stop procedure here as it's an execution that's not currently running but it's marked as running in DB
					} else {
					  // If active execution could not be found check if it is a waiting one
					  returnData = await this.waitTracker.stopExecution(executionId);
					}
				} else {
					returnData = {
						mode: result.mode,
						startedAt: new Date(result.startedAt),
						stoppedAt: result.stoppedAt ? new Date(result.stoppedAt) : undefined,
						finished: result.finished,
						status: result.status,
					};
				}
```

**Review by krynble — CHANGES_REQUESTED** [review]

**Review by krynble — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.233.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.233.0)
